_exercises/no/08-exporting-your-track/01-exporting-your-track.md

```diff
@@ -0,0 +1,81 @@
+---
+chapter: Eksportere musikken din
+title: Eksportere musikken din
+lang: no
+layout: exercise
+---
+
+Når du er fornøyd med det første sporet ditt vil du kanskje lagre sporet som en lydfil og dele det med vennene dine online (for eksempel på din egen <a href="http://www.soundcloud.com/mehackit">SoundCloud</a> side). Først må du ta opp sangen din ved å bruke 'record' funksjonen i Sonic Pi. 
+
+Stegene for å ta opp er som følger:
+
+* Klikk på **Rec** knappen
+* Når du ser at Rec er trykket inn kan du starte sangen din ved å trykke på **Run**
+* Når sangen er ferdig eller du har spilt mange nok ganger gjennom live-løkkene dine trykk på **Stop**
+* Til slutt trykker på du **Rec** igjen for å stoppe opptaket
+* Lagre lydfilen din i diaologvinduet som spretter opp (for eksempel med navnet *MinSang.wav*)
+
+{% include videoplayer.html filepath="/assets/video/record_audio" %}
+
+Når du har lagret lydfilen din er du klr for å spille og dele den. Vi anbefaler at du fikser litt på lyden for å få den til og høres proffere og høyere ut først. For å gjøre dette bruker vi *Audacity* som er gratis og åpen kildekode programvare for å ta opp og redigere lyd. Du kan laste ned *Audacity* (for Windows, Linux og OS X) her: <a href="http://www.audacityteam.org/download/">http://www.audacityteam.org/download/</a>. 
```

Comment:
For å gjøre dette bruker vi *Audacity* som er programvare for å ta opp og redigere lyd. Audacity er gratis og har åpen kildekode.